content/en/docs/reference/scheduling/config.md

```diff
@@ -1,9 +1,17 @@
 ---
-title: Scheduling Profiles
+title: Scheduling Configuration
```

Comment:
If renaming a page, remember to add a redirect.

Reply:
will do

Reply:
Can you refer me to an example of how to do a redirect?

Reply:
Done